lib/assets/javascripts/src/local-time/start.coffee

```diff
@@ -25,5 +25,8 @@ LocalTime.start = ->
     else
       nextFrame(startController)
 
+LocalTime.processing = ->
+  LocalTime.getController().started
```

Comment:
This may not be necessary as consumers can now call .start() repeatedly without worrying if LocalTime is processing.

https://github.com/basecamp/local_time/pull/153